spacy/tests/serialize/test_serialize_kb.py

```diff
@@ -139,6 +159,6 @@ def custom_kb_factory(vocab):
         nlp2 = util.load_model_from_path(tmp_dir)
         entity_linker2 = nlp2.get_pipe("entity_linker")
         # After IO, the KB is the standard one
-        assert type(entity_linker2.kb) == InMemoryLookupKB
+        assert type(entity_linker2.kb) == SubInMemoryLookupKB
         assert entity_linker2.kb.entity_vector_length == 342
-        assert not hasattr(entity_linker2.kb, "custom_field")
+        assert hasattr(entity_linker2.kb, "custom_field")
```

Comment:
I would repeat the same `assert` as before: 
```
assert entity_linker.kb.custom_field == 666
```

Reply:
Updated in https://github.com/explosion/spaCy/pull/12320/commits/b6d9f4c5ab7ebcf8a3845835de8b142692cba2fe.